src/Symfony/Component/ExpressionLanguage/Node/Node.php

```diff
@@ -75,4 +75,27 @@ public function evaluate($functions, $values)
 
         return $results;
     }
+
+    public function dump()
+    {
+        return sprintf('<Dumping a "%s" instance is not supported yet>', get_class($this));
```

Comment:
I don't know. It was easier to be able to see a "preview" while developing than an exception.
But indeed, as the expression is not valid, let throw an expression. (done)